rules/windows/process_creation/proc_creation_win_sysinternals_sysmon_config_update.yml

```diff
@@ -0,0 +1,28 @@
+title: Sysmon Configuration Update
+id: 87911521-7098-470b-a459-9a57fc80bdfd
+status: test
+description: Detects updates to Sysmon's configuration. Attacker might update or replace Sysmon configuration with a bare bone one to avoid monitoring without shutting down with the service permanently
```

Comment:
```suggestion
description: Detects updates to Sysmon's configuration. Attackers might update or replace the Sysmon configuration with a bare bone one to avoid monitoring without shutting down the service completely
```